Blish HUD/GameServices/Input/Keyboard/KeyboardHandler.cs

```diff
@@ -229,7 +229,7 @@ public class KeyboardHandler : IInputHandler {
                         // If we found an active window, close it
                         var activeWindow = WindowBase2.ActiveWindow;
 
-                        if (activeWindow != null && activeWindow.CanClose) {
+                        if (activeWindow != null && activeWindow.CanCloseWithEscape) {
```

Comment:
This likely still needs the check for CanClose.  This change would allow windows with this set to false still able to close with ESC.

`if (activeWindow != null && activeWindow.CanClose && activeWindow.CanCloseWithEscape)) {`